docs/cloud/alerts-notifications/notifications.md

```diff
@@ -1,74 +1,54 @@
-# Cloud alert notifications
+# Centralized Cloud Notifications Reference
 
-import Callout from '@site/src/components/Callout'
-
-Netdata Cloud can send centralized alert notifications to your team whenever a node enters a warning, critical, or
-unreachable state. By enabling notifications, you ensure no alert, on any node in your infrastructure, goes unnoticed by
-you or your team.
+Netdata Cloud can send centralized alert notifications to your team whenever a node enters a warning, critical, or unreachable state. By enabling notifications, you ensure no alert, on any node in your infrastructure, goes unnoticed by you or your team.
 
 Having this information centralized helps you:
+
 * Have a clear view of the health across your infrastructure, seeing all alerts in one place.
-* Easily [set up your alert notification process](https://github.com/netdata/netdata/blob/master/docs/cloud/alerts-notifications/manage-notification-methods.md): 
+* Easily [set up your alert notification process](https://github.com/netdata/netdata/blob/master/docs/cloud/alerts-notifications/manage-notification-methods.md):
 methods to use and where to use them, filtering rules, etc.
 * Quickly troubleshoot using [Metric Correlations](https://github.com/netdata/netdata/blob/master/docs/cloud/insights/metric-correlations.md)
 or [Anomaly Advisor](https://github.com/netdata/netdata/blob/master/docs/dashboard/anomaly-advisor-tab.md)
 
-If a node is getting disconnected often or has many alerts, we protect you and your team from alert fatigue by sending
-you a flood protection notification. Getting one of these notifications is a good signal of health or performance issues
-on that node.
-
-Admins must enable alert notifications for their [Space(s)](https://github.com/netdata/netdata/blob/master/docs/cloud/alerts-notifications/manage-notification-methods.md#manage-space-notification-settings). All users in a
-Space can then personalize their notifications settings from within their [account
-menu](https://github.com/netdata/netdata/blob/master/docs/cloud/alerts-notifications/#manage-user-notification-settings).
-
-<Callout type="notice">
+If a node is getting disconnected often or has many alerts, we protect you and your team from alert fatigue by sending you a flood protection notification. Getting one of these notifications is a good signal of health or performance issues on that node.
```

Comment:
```suggestion
To prevent alert overload, we automatically send a "flood protection" notification if a node experiences frequent disconnections or triggers a high volume of alerts. This notification highlights potential health or performance problems on that specific node.
```